【发现已存在的游戏窗口】
正在返回菜单,请保持游戏画面处于前台
正在返回菜单,请保持游戏画面处于前台
【现在处于主菜单】
【目标出现】images/lobby.png 置信度是：0.999919593334198
【1】等待目标中 images/old_guide.png
【目标出现】images/old_guide.png 置信度是：0.9999997019767761
【1】等待目标中 images/map_sixcent.png
【2】等待目标中 images/map_sixcent.png
【3】等待目标中 images/map_sixcent.png
【4】等待目标中 images/map_sixcent.png
【5】等待目标中 images/map_sixcent.png
【6】等待目标中 images/map_sixcent.png
【7】等待目标中 images/map_sixcent.png
【8】等待目标中 images/map_sixcent.png
【9】等待目标中 images/map_sixcent.png
【10】等待目标中 images/map_sixcent.png
【11】等待目标中 images/map_sixcent.png
【12】等待目标中 images/map_sixcent.png
【13】等待目标中 images/map_sixcent.png
【14】等待目标中 images/map_sixcent.png
【15】等待目标中 images/map_sixcent.png
-------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\gelon\Desktop\Pytutorial\ZenlessZoneZero_Automation\ZA_Main.py", line 718, in run
    coffee()
    ~~~~~~^^
  File "C:\Users\gelon\Desktop\Pytutorial\ZenlessZoneZero_Automation\ZA_Main.py", line 382, in coffee
    scroll_move('images/map_sixcent.png')
    ~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gelon\Desktop\Pytutorial\ZenlessZoneZero_Automation\ZA_Main.py", line 80, in scroll_move
    match_tpl(tpl,'scroll_move',skip=skip)
    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gelon\Desktop\Pytutorial\ZenlessZoneZero_Automation\ZA_Main.py", line 72, in match_tpl
    time.sleep(wait_speed)
    ~~~~~~~~~~^^^^^^^^^^^^
KeyboardInterrupt

Exception:主线程出错
按回车键退出(Enter)Traceback (most recent call last):
  File "C:\Users\gelon\Desktop\Pytutorial\ZenlessZoneZero_Automation\ZA_Main.py", line 718, in run
    coffee()
    ~~~~~~^^
  File "C:\Users\gelon\Desktop\Pytutorial\ZenlessZoneZero_Automation\ZA_Main.py", line 382, in coffee
    scroll_move('images/map_sixcent.png')
    ~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gelon\Desktop\Pytutorial\ZenlessZoneZero_Automation\ZA_Main.py", line 80, in scroll_move
    match_tpl(tpl,'scroll_move',skip=skip)
    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gelon\Desktop\Pytutorial\ZenlessZoneZero_Automation\ZA_Main.py", line 72, in match_tpl
    time.sleep(wait_speed)
    ~~~~~~~~~~^^^^^^^^^^^^
KeyboardInterrupt

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\gelon\Desktop\Pytutorial\ZenlessZoneZero_Automation\ZA_Main.py", line 744, in <module>
    run()
    ~~~^^
  File "C:\Users\gelon\Desktop\Pytutorial\ZenlessZoneZero_Automation\ZA_Main.py", line 741, in run
    input('Exception:主线程出错\n按回车键退出(Enter)')
    ~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen codecs>", line 325, in decode
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xff in position 0: invalid start byte
